src/ui/link/index.tsx

```diff
@@ -71,7 +71,7 @@ const Link = forwardRef<HTMLButtonElement | HTMLAnchorElement, LinkProps>(
               generateLocationFromPath(route.path, routeAttributes)
             )) ||
           ''
-        : to;
+        : `${routeAttributes.basePath}${to}`;
```

Comment:
I'm not sure if there's a better way to handle this. It's essentially the same logic as `generateLocationFromPath` except this should already by a generated path, thus no transformations.